### ISSUE TITLE
Search input on popup

### ISSUE THREAD

**Issue by ans1genie** [dvdvdmt/popup-tab-switcher#15]
When: the user opens the popup
Then: A search input on top of the list to search for tab using keywords

**Comment by dnknn** [comment]
Searching is not necessary. This is a switcher, not a manager.
The core function is similar to Alt+Tab switching, Only the most **recent** tabs will be used,
About the function of the tabs, there are too many, if there are, then PTS is not a switcher.

**Comment by dkjain** [comment]
> Searching is not necessary. This is a switcher, not a manager.
> The core function is similar to Alt+Tab switching, Only the most **recent** tabs will be used,
> About the function of the tabs, there are too many. if `PTS` also have , then `PTS` is not a switcher.

I don't agree that by adding the search input field it would become a manager, in-fact it just another way for user to provide input to switch to the desired tab. I would really like to see this implemented.

Thanks
